builds/e2e/nested-connectivity.yaml

```diff
@@ -82,6 +82,7 @@ stages:
       identityServiceArtifactName: packages_ubuntu-18.04_amd64
       identityServicePackageFilter: aziot-identity-service_*_amd64.deb   
       artifactName: iotedged-ubuntu18.04-amd64
+      upstream.protocol: 'mqtt'
```

Comment:
This should be hardcoded, not fed through the pipeline. I tried to add a stage level environment variable for further consolidation (i.e. line 21), however devops wouldn't parse the yaml.